tests/dialogBoxes.spec.ts

```diff
@@ -10,7 +10,7 @@ test.describe('Dialog boxes', () => {
         await page.getByRole('link', { name: 'Pet Types' }).click()
         await expect(page.getByRole('heading')).toHaveText('Pet Types')
         await page.getByRole('button', { name: "Add" }).click()
-        await expect(page.locator('h2').nth(1)).toHaveText('New Pet Type')
+        await expect(page.locator('app-pettype-add')).toContainText('New Pet Type');
```

Comment:
This is a validation that the section contains text 'New Pet Type'.
We need validation that a header of this section is equal to 'New Pet Type'